FreeTubeApp/FreeTube
# Fix comment section style inconsistency

# Fix comment section style inconsistency




## Pull Request Type


## Related issue
N/A

## Description
After all comments loaded there is no spacing at the bottom
It's very obvious when there are only a few comments

## Screenshots 

<details>
<summary>Before</summary>

![image](https://user-images.githubusercontent.com/1018543/221727724-e2344a83-d5b5-4cae-a11a-c2fafa30ea95.png)
![image](https://user-images.githubusercontent.com/1018543/221727754-ec911383-4feb-4658-943a-47975bf68677.png)
![image](https://user-images.githubusercontent.com/1018543/221727770-6b0d4877-df6f-4b95-93f9-147b6f54f2ee.png)
![image](https://user-images.githubusercontent.com/1018543/221730029-1a8e3686-c7ff-46e5-890c-9e7e6b8717c6.png)

</details>

<details>
<summary>After</summary>

![image](https://user-images.githubusercontent.com/1018543/221727826-65846991-d823-400c-8e1f-957e2f0a98fd.png)
![image](https://user-images.githubusercontent.com/1018543/221727841-c90d4ae1-0316-48ac-a0f4-8c4ec797e4f6.png)
![image](https://user-images.githubusercontent.com/1018543/221727856-f994f72a-2531-4e26-871e-d8e0189dc07e.png)
![image](https://user-images.githubusercontent.com/1018543/221730055-65d61362-d878-4ba5-a8ed-476aceeda2c7.png)

</details>

## Testing 



A. Video with comments
- View a video with comments (e.g. https://www.youtube.com/watch?v=XDEmGBC7p68&list=PLzFTGYa_evXi8n4pPFBUs1sIPFZJmK7Rq)
- Inspect comment section style before comments loaded
- Inspect comment section style after comments loaded (when not all comments are loaded)
- Inspect comment section style after comments loaded (when all comments are loaded)

B. Video without comment
- View a without comment (e.g. https://youtu.be/CIhSi6gwT9E)
- Inspect comment section style before comments loaded
- Inspect comment section style after comments loaded
- Inspect visibility of components inside comments section (i.e. only "no comments" message should be visible)



## Desktop

- **OS:** MacOS
- **OS Version:** 12.6.2
- **FreeTube version:** 3c3325dcf0f0f9b2c8aefdbf52d610b778c10f5e

## Additional context

Side effect:
- Comment section becomes "fatter" before comments loaded
